### PR TITLE
Update General Elment and URL

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/88215

### CODE REVIEW

File: EnglishFilter/sections/general_url.txt [hunk 1]
```diff
@@ -81,6 +81,7 @@ porn*.net/bees.js|
 !+ NOT_OPTIMIZED
 /getads?
 !
+/bannerpop/uploads/*$image,script
```
Comment (piquark6046) [inline EnglishFilter/sections/general_url.txt:84]: This rule can apply to many website.
https://www.todaykorea.co.kr/
https://www.econovill.com/ ( https://www.econovill.com/news/articleView.html?idxno=540458 )
https://datanet.co.kr/
https://www.entermedia.co.kr/ ( https://www.entermedia.co.kr/news/articleView.html?idxno=27002 )
https://www.sisafocus.co.kr/news/articleView.html?idxno=262932
http://www.newsian.co.kr/ ( http://www.newsian.co.kr/news/articleView.html?idxno=49785 )
http://www.constimes.co.kr/
https://www.idsn.co.kr/ ( https://www.idsn.co.kr/news/articleView.html?idxno=50752 )
https://www.ggilbo.com ( https://www.ggilbo.com/news/articleView.html?idxno=858021 )
http://www.andongdaily.com/ ( http://www.andongdaily.com/news/articleView.html?idxno=24475 )
https://www.tiptipnews.co.kr/news/articleView.html?idxno=80869
http://www.fntoday.co.kr/ ( http://www.fntoday.co.kr/news/articleView.html?idxno=260316 )

File: EnglishFilter/sections/general_elemhide.txt [hunk 2]
```diff
@@ -97,6 +97,7 @@
 ##.div-gpt-ad:not([style^="width: 1px; height: 1px;"])
 ##[class^="div-gpt-ad"]:not([style^="width: 1px; height: 1px;"])
 !
+##div[class^="banner_"]
```
Comment (piquark6046) [inline EnglishFilter/sections/general_elemhide.txt:100]: This rule also can apply to many website.
https://www.bosa.co.kr/news/articleView.html?idxno=2154214
https://www.econovill.com/ ( https://www.econovill.com/news/articleView.html?idxno=540458 )
https://www.sisafocus.co.kr/news/articleView.html?idxno=262932
http://www.newsian.co.kr/ ( http://www.newsian.co.kr/news/articleView.html?idxno=49785 )
https://www.idsn.co.kr/ ( https://www.idsn.co.kr/news/articleView.html?idxno=50752 )
https://www.ggilbo.com ( https://www.ggilbo.com/news/articleView.html?idxno=858021 )
http://www.andongdaily.com/ ( http://www.andongdaily.com/news/articleView.html?idxno=24475 )
https://www.tiptipnews.co.kr/news/articleView.html?idxno=80869
http://www.fntoday.co.kr/ ( http://www.fntoday.co.kr/news/articleView.html?idxno=260316 )

Comment (Alex-302) [reply EnglishFilter/sections/general_elemhide.txt:100]: This rule is extremely dangerous) Please make domain specific.

File: EnglishFilter/sections/general_url.txt (hunk 1 above)
Comment (Alex-302) [inline EnglishFilter/sections/general_url.txt:84]: It seems modifiers are unnecessary in this case. Just `/bannerpop/uploads/*`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_url.txt (hunk 1 above)
Comment (piquark6046) [reply EnglishFilter/sections/general_url.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/88218/commits/2023e9a687a9d5b40c3d189ee58ecddc5ff55488

**Comment by piquark6046** [comment]
Oh, I forgot leaving `Co-authored-by` in https://github.com/AdguardTeam/AdguardFilters/pull/88218/commits/2023e9a687a9d5b40c3d189ee58ecddc5ff55488 commit.

**Comment by piquark6046** [comment]
@Alex-302 What about this rule instead of https://github.com/AdguardTeam/AdguardFilters/pull/88218/files#diff-fbf56167647d1d9892a9a9ccdd71a6c88bdf70c0d33822f99e0270e936c4b898R4350-R4351 ?
`##div.banner_box`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**